TERMSOFSERVICE.md

```diff
@@ -16,9 +16,7 @@ performance, a person or a location.
 represent the exclusive basis for all legal relationships established between
 us and you, related to the use of Tideways.
 
-1.4. By accessing or using Tideways, you are agreeing to these ToS and
-concluding a legally binding contract with us. Do not access or use Tideways if
-you are unwilling or unable to be bound by the ToS.
+1.4. [Removed]
```

Comment:
We have always asked explicitly for your consent to the ToS when creating an account with Tideways. This paragraph was therefore never needed.